media/server/gstplayer/source/GstPlayer.cpp

```diff
@@ -390,9 +393,16 @@ GstBuffer *GstPlayer::createDecryptedBuffer(const IMediaPipeline::MediaSegment &
         }
         GstBuffer *subsamples = m_gstWrapper->gstBufferNewWrapped(subsamplesRaw, subsamplesRawSize);
 
-        m_decryptionService.decrypt(mediaSegment.getMediaKeySessionId(), gstBuffer, subsamples,
-                                    mediaSegment.getSubSamples().size(), initVector, keyId,
-                                    mediaSegment.getInitWithLast15());
+        GstStructure *info = m_gstWrapper->gstStructureNew("application/x-cenc", "encrypted", G_TYPE_BOOLEAN, TRUE,
```

Comment:
Nit pick: I think that "encrypted" field is not needed since protection_meta will be present only when data is encrypted.
Name of the structure ("application/x-cenc") might be a little bit missleading, because we might want to use various enc scheme. Since only decrypter will use it, it can be any string, right?

Reply:
Agreed about the encrypted field, but I _think_ the name is probably actually a good change because of the upcoming OCDM changes to support cbcs. Currently only cenc is supported (as I understand it) and with the incoming changes we will introduce support for “application/x-cbcs” as well.

Reply:
I've changed the name to of the structure 'decryption_metadata' to make it more generic, we can add a new property to store the encryption scheme when we support multiple.